index.rst

```diff
@@ -118,68 +153,81 @@ to diagnose or characterize certain behaviour), and personal notebooks. The deta
 scope of this technote. Until the re-organisation is completed, tags will be made of the repo at least every ~6 months,
 after which all files larger than 20 MB (TBR) or older than 1 year will be deleted from the develop branch.
 
+.. note::
+
+    Another suggestion is to use git-lfs for large files in the repo. This needs further
```

Comment:
is the idea to use git-lfs for notebooks? I am not sure we will ever get notebooks that are large enough that will require that if we clean them up.